pages/rdm_cycle/processing.md

```diff
@@ -1,36 +1,36 @@
 ---
 title: Processing
-keywords:
-contributors: [Rob Hooft]
+keywords: [Data Cleaning, Data Formatting, Quality Check, Data validation]
```

Comment:
Should we use only lower case @bedroesb ?

Reply:
good question, it is not case sensitive, so it is up to us to decide. But if we decide we should put it in the style guide and make it conform everywhere.

Reply:
I think we use only lower case and space between the words in the other pages.

Reply:
Changed to lower case!